src/tcp.c

```diff
@@ -450,6 +450,28 @@ tcp_get_ip_from_str(const char *src, struct sockaddr *sa)
 /**
  *
  */
+int
+tcp_get_sockaddr(struct sockaddr *sa, const char *s)
```

Comment:
Why you don't use already prepared tcp_get_ip_from_str() function which does the exactly similar thing ?